cypress/e2e/MovieListPage.cy.ts

```diff
@@ -0,0 +1,54 @@
+describe('Movie List Page', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000')
```

Comment:
It is better to move the base url into config instead of hardcoding it. Consider the scenario when your 3000 port is busy with something else and the app is launched on 3001 instead. In this case, one has to change every single line that includes 3000.